packages/account/src/predicate/predicate.ts

```diff
@@ -118,8 +120,7 @@ export class Predicate<TInputData extends InputValue[]> extends Account {
     /** Tx Params */
     txParams: TxParamsType = {}
   ): Promise<TransactionRequest> {
-    const request = await super.createTransfer(destination, amount, assetId, txParams);
-    return this.populateTransactionPredicateData(request);
+    return super.createTransfer(destination, amount, assetId, txParams);
```

Comment:
We can remove this method from `Predicate` altogether now as it's doing nothing additional.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2021/commits/6cc858b577fcef20351267211d2ee1c3017522f3